apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/write/WriteBridgeService.java

```diff
@@ -592,8 +593,19 @@ private Uni<ResultSet> executeBatch(
                             .setValue(queriesConfig.consistency().writes())));
     batchQueries.forEach(batch::addQueries);
     Batch batchBuilt = batch.build();
+    if (batchBuilt.getSerializedSize() >= BATCH_PAYLOAD_SIZE_LIMIT) {
+      logger.warn(
+          "Tenant: {}, Batch payload size : {} in bytes, Number of CQL Statements : {}",
+          requestInfo.getTenantId().orElse(null),
+          batchBuilt.getSerializedSize(),
+          batchBuilt.getQueriesCount());
+    }
     if (logger.isDebugEnabled()) {
```

Comment:
Should be `} else {` so we wont log twice?

Reply:
Fixed in this commit [e9535a9](https://github.com/stargate/stargate/pull/2932/commits/e9535a9818edcf3e53784f9ad21b544e15b8f785)